### ISSUE TITLE
[API-related bug] Error while opening specific chapter in specific manga

### ISSUE THREAD

**Issue by DjMike238** [inorichi/tachiyomi#2120]
**App version:** 0.8.4

**Android version:** LineageOS 16.0 (Android 9)

**Issue:** Error while opening specific chapter in specific manga

**Steps to reproduce**

 1. Open manga "Tomo-chan Is a Girl!" from Manga Rock (EN)
 2. Try to open chapter 938 (latest one at the time of writing this)
 3. Get the error in the screenshot below:

![](https://i.imgur.com/yLXCW1J.png)

**Comment by arkon** [comment]
Works fine for me. Can you open it in the webview?

**Comment by DjMike238** [comment]
How do I do that?

**Comment by DjMike238** [comment]
I tried to open it in webview, but it just keeps loading without showing me anything.
Same thing with other chapters, which loaded just fine outside the webview.

**Comment by DjMike238** [comment]
Little update: I took a look at Tachiyomi's code, and tried to do some API requests using Postman.

Turns out that if I request the pages of the 937th chapter, it gives me an URL for a .mri file (judging by the code, is an encrypted webm of the actual page), but if I request the pages of the 938th chapter, it says "Chapter is removed". Weird.

Here's the URL for the request to obtain that manga's information:
`https://api.mangarockhd.com/query/web401/info?oid=mrs-serie-254590&country=`

Here's the URL for the request of the 938th chapter:
`https://api.mangarockhd.com/query/web401/pages?oid=mrs-chapter-200082835`
